ROMFS/px4fmu_common/init.d/airframes/4014_s500

```diff
@@ -12,20 +12,9 @@
 #
 
 . ${R}etc/init.d/rc.mc_defaults
+. ${R}etc/init.d/rc.quad_x_default_geometry
 
 param set-default MC_ROLLRATE_P 0.18
 param set-default MC_PITCHRATE_P 0.18
 param set-default MC_ROLLRATE_I 0.15
 param set-default MC_PITCHRATE_I 0.15
-
-param set-default CA_ROTOR_COUNT 4
```

Comment:
One thing to consider is that people will create their own airframe by forking the closest existing example and modifying it. Directly laying out all relevant parameters makes that as obvious as possible.

I'd like to get to the point where airframes are literally declarative files that we can validate statically.

For flash savings in general there are some things I believe we need to be pursuing fundamentally differently that will yield orders of magnitude larger savings for the way people actually want/need to use PX4.

Reply:
So separate builds yes.

Is there a way to compress the ROMFS? That would anyways save better than the manual work.

Reply:
>One thing to consider is that people will create their own airframe by forking the closest existing example and modifying it. Directly laying out all relevant parameters makes that as obvious as possible.

We already had the rc.mc_defaults etc that we included, so this change here is just following up on that. And it really also reduces repetitive lines that add chances for error. For me that together with the flash savings overweights the small complexity increase. 

>For flash savings in general there are some things I believe we need to be pursuing fundamentally differently that will yield orders of magnitude larger savings for the way people actually want/need to use PX4.

Could you note them down eg in an Issue or somewhere? For us flash constraints are a big pain at the moment, so let's get the work rolling!